lessson_3_object_array_map_set/constructor_new/constructor_new_1.js

```diff
@@ -0,0 +1,15 @@
+// https://learn.javascript.ru/task/two-functions-one-object
+// Возможно ли создать функции A и B, чтобы new A() == new B()?
+let obj = {}; // my code
+function A() {
+	return obj;
+}
+function B() {
+	return obj;
```

Comment:
Please, use only one tabulation for functions: 

`function A() {
    return obj;
}`